express-intro/server.ts

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+
+const app = express();
+
+app.listen(8080, () => {
```

Comment:
Although it doesn't matter, it's customary to put the `listen` command as the very last one in the file. It makes the code read like "First set everything up, then listen."